app/src/main/java/io/pumpkinz/pumpkinreader/model/Comment.java

```diff
@@ -13,13 +15,17 @@
     Comment parentComment;
     List<Comment> childComments;
 
-    public Comment() {}
+    public Comment() {
+        this.kids = new ArrayList<>();
+        this.childComments = new ArrayList<>();
```

Comment:
Yes, if not, we have to define our own JSON parser.
What if we change the getter instead? (`getKids` into `getCommentIds`)